libraries/chain/include/eosio/chain/authority_checker.hpp

```diff
@@ -109,6 +117,11 @@ namespace detail {
          {}
 
          bool satisfied(const permission_level& permission, uint16_t depth = 0) {
+            // QUESTION: Should we not catch the chain::permission_query_exception from permission_to_authority and try to
```

Comment:
I vote "yes" we treat failure from `permission_to_authority` callback as if `satisfied` returned `false`.

Reply:
Done. Changes in commit 687e6ea.